ArjunaJTS/jtax/tests/classes/com/hp/mwtests/ts/jta/jts/jca/XATerminatorImpleUnitTest.java

```diff
@@ -31,12 +31,40 @@
 
 package com.hp.mwtests.ts.jta.jts.jca;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
it would be better if the imports could be kept unchanged if possible (at least just the additions/subtractions)